src/components/Nav/Nav.scss

```
@@ -0,0 +1,205 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+.nav {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  height: 70px;
+  color: #222222;
+  border-bottom: #eeeeee 1px solid;
+  font-weight: 700;
+
+  .nav_menu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 1440px;
+
+    .header_logo {
+      flex: 1;
+      .leader_logo_image {
+        height: 33px;
+        padding-right: 35px;
+        cursor: pointer;
+      }
+    }
+
+    .header_shop_nav {
+      display: flex;
+      flex: 10;
+
+      .nav_underLine {
+        position: relative;
+        margin: 0 35px;
+        font-family: 'Roboto', sans-serif;
+        line-height: 70px;
+        cursor: pointer;
+
+        .header_sub_nav {
+          display: none;
+        }
+
+        &:hover .header_sub_nav {
+          position: absolute;
+          display: flex;
+          align-items: center;
+          height: 70px;
+          top: 100%;
+          background-color: #222222;
+          color: white;
+          z-index: 2;
+        }
+      }
+
+      li:before {
+        content: '';
+        position: absolute;
+        height: 4px;
+        width: 0;
+        bottom: 20px;
+        background-color: #fdcf41;
+        transition: 0.3s;
+      }
+
+      li:hover:before {
+        width: 100%;
+      }
+    }
+
+    .header_user_nav {
+      position: relative;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      flex: 4;
+      font-family: 'Roboto', sans-serif;
+      cursor: pointer;
+
+      .header_search,
+      .heder_alert,
+      .header_cart,
+      .header_user {
+        img {
+          width: 24px;
+          transition: 200ms cubic-bezier(0.4, 0, 0.2, 1);
+        }
+      }
+
+      .header_search,
+      .heder_alert,
+      .header_cart,
+      .header_user {
+        img:hover {
+          width: 24px;
+          filter: opacity(0.5) drop-shadow(0 0 0 #fdcf41);
+        }
+      }
```

```suggestion
      .header_search,
      .heder_alert,
      .header_cart,
      .header_user {
        img {
          width: 24px;
          transition: 200ms cubic-bezier(0.4, 0, 0.2, 1);
         &: hover {
            width: 24px;
            filter: opacity(0.5) drop-shadow(0 0 0 #fdcf41);
            }
         }
      }
```

* sass에서 & 은 부모선택자 입니다. 
- sass에서 nesting을 한 경우에 &를 쓰면 부모의 className이 그대로 적용됩니다.
- 이러한 문법을 이용해서 편리하게 selector를 사용할 수 있습니다.
- 선택자를 중복해서 선언하지 않아도 됩니다.
- 부모선택자는, 가상선택자나, 가상요소에도 사용 됩니다.

---

넵 알겠습니당